src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -62,7 +62,6 @@
           >
             <div class="d-flex flex-column flex-sm-row justify-center align-center">
               <img
-                slot-scope=""
```

Comment:
This was giving us a warning

Reply:
This was auto added by eslint. Looks like it wasn't doing much anyway :)